src/firebase-social-provider.js

```diff
@@ -64,12 +65,10 @@ FirebaseSocialProvider.prototype.login = function(loginOpts) {
         };
 
         this.setPresence_(true);
+        this.detectDisconnect_();
```

Comment:
Kind of a nit, but maybe a name like "setupDetectDisconnect" or such would be good, since we're (hopefully) not detecting a disconnect right now but just setting up the machinery for the possibility.